gbpservice/nfp/configurator/modules/configurator.py

```diff
@@ -514,6 +519,24 @@ def nfp_module_post_init(sc, conf):
     try:
         cm = get_configurator_module_instance(sc)
         cm.init_service_agents_complete(sc, conf)
+
+        #TODO:Need to generalize the following code in library.
+        context =  n_context.Context('configurator', 'configrator')
```

Comment:
We should add this code in a different function. Don't fix this now. Lets do this in next PR.